action.yml

```diff
@@ -30,4 +30,4 @@ inputs:
     default: "true"
 runs:
   using: "docker"
-  image: docker://public.ecr.aws/p4v7w0a5/lazy/dotnet-action:v1.5.0
\ No newline at end of file
+  image: docker://public.ecr.aws/p4v7w0a5/lazy/dotnet-action:v1.6.0
```

Comment:
Why is this v1.6.0. This should be commit sha for branch testing. Updating with minor tag will create discrepancies  when someone creates a tag with same version. Not sure if we have enabled image mutability in case of same tag.

Reply:
During standup, I stated that I had updated the tag to move things along. The testing was done with `image: docker://public.ecr.aws/p4v7w0a5/lazy/dotnet-action:sha-26be73c40fa28d79160a72ef46b08e4b4c266ff0`. `v1.6.0` will be the new tag when merged. I will roll it back

The commit: https://github.com/variant-inc/actions-dotnet/pull/32/commits/bec7e04541423e14690b9595bc9b0cda194a68ab